index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <link rel="stylesheet" href="assets/css/css.css">
+    <link rel="stylesheet" href="assets/css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+    <section>
+        <div class="container">
+            
+            <h4>One Account.</h4>
+            <h5> Everything Intuit.</h5>
+            <p>Create and account for all intuits's products</p> 
+            <p>including Mint.<a href="#" > Learn More</a></p>
+
+            <form> 
+                <label>Email address </label><br>
+                <input>
+                <br>
+                <label id="phone-label">Phone (Recommended) </label><br>
+                <input>
+                <br>
+                <label for="Password">Password</label><br>
+                <input type="password" name="password" id="password"><br>
+                <input type="submit" value="Create Account" id="submit">
+
+                <p> <i> By clicking Create Account, you agree to our</p>
+                <p><a href="#">Terms of service</a> and have read and ackowledge</p>
+                <p>our <a href="#">Privacy Statement</a></i></p>
```

Comment:
Element "p" not allowed as child of the element "i" in this context. (Suppressing further errors from this subtree.)